elasticizer/pipelines/med_search.py

```diff
@@ -0,0 +1,186 @@
+import elasticizer
+
+fields = [
+  'id_cfpb',
+  'name',
+  'name_short',
+  'active_date',
+  'inactive_date',
+  'inst_type_code',
+  'inst_status_code',
+  'fax_number',
+  'phone_number'
+  ]
+
+# -----------------------------------------------------------------------------
+# Tasks
+# -----------------------------------------------------------------------------
+
+class Extract(PostgresQuery):
+    host = os.getenv('DB_HOST', '127.0.0.1')
+    database = os.getenv('DB_NAME', 'mydb')
+    user = os.getenv('PGUSER', 'vagrant')
+    password = os.getenv('PGPASSWORD', 'vagrant')
+    table = 'institution'
+    query = 'SELECT * FROM institution'
+
+
+class Format(luigi.Task):
+    index = luigi.Parameter()
+
+    def _projection(self, row):
+        return {x[0]:x[1] if 'date' not in x[0] else x[1].isoformat()
+                for x in zip(fields, row)}
+
+    def requires(self):
+        return [Extract()]
+ 
+    def output(self):
+        return luigi.LocalTarget(self.index + "-formatted.json")
+ 
+    def run(self):
+        extractor = self.input()[0]
+
+        template = "SELECT {0} FROM {1};"
+        sql = template.format(', '.join(fields), extractor.table)
+
+        with extractor.connect().cursor() as cur:
+            cur.execute(sql)
+            data = [self._projection(row) for row in cur]
+            with self.output().open('w') as f:
+                json.dump(data, f)
+
+class ValidMapping(luigi.Task):
+    index = luigi.Parameter()
+
+    def requires(self):
+        return []
+
+    def output(self):
+        return luigi.LocalTarget(self.index + "-mapping.json")
+
+    def run(self):
+        mappings = {
+          "_source": {
+            "excludes": [
+              "autocomplete",
+              "suggest"
+            ]
+          },
+          "_all": {
+            "type": "string",
+            "index_analyzer": "standard",
+            "search_analyzer": "standard"
+          },
+          "_id": {
+            "path": "id_cfpb"
+          },
+          "properties": {
+            "phonetic": {
+              "type": "string",
+              "analyzer": "phonetic",
+              "store": True
+            },
+            "autocomplete": {
+              "type": "string",
+              "index_analyzer": "shingle_analyzer",
+              "search_analyzer": "standard"
+            },
+            "suggest": {
+              "type": "completion",
+              "index_analyzer": "standard",
+              "search_analyzer": "standard",
+              "payloads": False
+            }
+          }
+        }
+        with self.output().open('w') as f:
+            json.dump(mappings, f)
+
+class ValidSettings(luigi.Task):
+    index = luigi.Parameter()
+
+    def requires(self):
+        return []
+
+    def output(self):
+        return luigi.LocalTarget(self.index + "-settings.json")
+
+    def run(self):
+        settings = {
+          "refresh_interval": "-1",
```

Comment:
@mehtadev17 I think that's the intention (for this and `ValidMapping`'s output), but for now it's a placeholder for code that will eventually validate those files.